augly/tests/image_tests/base_unit_test.py

```diff
@@ -96,13 +96,17 @@ def evaluate_class(
         transform_class: Callable[..., Image.Image],
         fname: str,
         metadata_exclude_keys: Optional[List[str]] = None,
+        check_mode: Optional[bool] = True
```

Comment:
```suggestion
        check_mode: Optional[bool] = True,
```
missing trailing comma